lib/screens/recipe_detail_screen.dart

```diff
@@ -10,22 +10,61 @@ class RecipeDetailScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      backgroundColor: Colors.black,
       appBar: AppBar(
-        title: Text(recipe.name),
+        backgroundColor: Theme.of(context).colorScheme.primary,
+        title: Text(
+          'EPAM Cocktail',
+          style: TextStyle(
+            color: Theme.of(context).colorScheme.background,
+            fontFamily: 'EPAM2',
+            fontWeight: FontWeight.w700,
+            fontSize: 30,
+          ),
+        ),
       ),
-      body: Padding(
+      body: SingleChildScrollView(
```

Comment:
Beacuse the if the text and image was greater than screen it would not scroll andgive exception, I do not know much about making it scrollable when text overflows screen